src/main/resources/config/application.yml

```diff
@@ -34,6 +34,10 @@ artemis:
     external-system-request:
         batch-size: 50  # wait the time below after 50 requests
         batch-waiting-time: 30000   # in ms = 30s
+    iosAppId: "2J3C6P6X3N.de.tum.cit.artemis"
+    androidAppPackage: "de.tum.informatics.www1.artemis.native_app.android"
+    androidSha256CertFingerprints: "D4:5C:58:77:B2:4E:2F:15:60:59:BB:C0:9C:1C:B0:A9:FD:3D:D8:F3:88:A8:B3:EA:DB:05:06:3B:7E:AF:0C:4F"
```

Comment:
Having proper values right now is fine, since that is the easiest way to test it on the test servers.

Before merging, I’d suggest removing the values and commenting them out since I assume this value is _very_ specific to the TUM instance and would not work anywhere else?

Reply:
So those values are not used to tell the app ‘this is _the_ Artemis-instance at TUM, open it in the app’, but rather a user configures the URL of their respective instance in the app, and the system uses the additional information to know that it is _some_ valid Artemis instance and links for that URL should be opened in the app?

In that case I misunderstood how it works, and keeping the values is fine. :+1: